Signal/src/ViewControllers/ConversationView/MessagesViewController.m

```diff
@@ -794,17 +794,17 @@ - (void)createBannerWithTitle:(NSString *)title bannerColor:(UIColor *)bannerCol
     [bannerView addSubview:closeButton];
     const CGFloat kBannerCloseButtonPadding = 8.f;
     [closeButton autoPinEdgeToSuperviewEdge:ALEdgeTop withInset:kBannerCloseButtonPadding];
-    [closeButton autoPinEdgeToSuperviewEdge:ALEdgeRight withInset:kBannerCloseButtonPadding];
```

Comment:
This was a bug in the custom leading/trailing methods.  Trailing margin constants need to be inverted.